tests/unit/test_regridding_service.py

```diff
@@ -139,3 +140,146 @@ def test_regrid_smap_data_end_to_end(
             assert 'longitude' in dt[group], f'failed: {group}'
             assert 'latitude' in dt[group], f'failed: {group}'
             assert 'altitude_dem' in dt[group], f'failed: {group}'
+
+
+def test_regrid_smap_excluded_variable_file(
+    test_spl3ftp_ncfile,
+    tmp_path,
+):
+    """Test the full regrid process with excluded variables.
+
+    This test adds a test varinfo config that excludes science variables:
+    "/.*altitude_dem.*"
+
+    Is it the same as the previous test, but the last assertion is that the
+    variable is not in the output.
+
+    """
+    input_filename = str(test_spl3ftp_ncfile)
+    output_filename = str(tmp_path / 'regridded_output.nc')
+    logger_mock = MagicMock()
```

Comment:
A genuine question: what was wrong with just creating a `Logger` instance? Is this to keep the test output nice and clean?

Reply:
Probably no reason.